examples/getting_started/react/README.md

```diff
@@ -0,0 +1,20 @@
+# Getting started
+
```

Comment:
It would be nice to have a one or two liner description of the app, what it does and what features/use cases of Offix it demonstrates.